docs/user-guide/line-plot.ipynb

```diff
@@ -248,6 +248,92 @@
     "        linestyle={'a': 'solid', 'b': 'dashed'},\n",
     "        marker=None)"
    ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "e485478c-3652-44fc-8b47-f368dad095cb",
+   "metadata": {},
+   "source": [
+    "## Bin edges\n",
+    "\n",
+    "When the coordinate of the data contains bin-edges, Plopp represents it as a step function instead of markers:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "32317d3d-2f05-417b-9004-8b0b047ab94a",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "da = pp.data.data_array(binedges=True)\n",
+    "pp.plot(da)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "c007556f-d0c2-4792-8ca9-0f5135d65a28",
+   "metadata": {},
+   "source": [
+    "## Masks\n",
+    "\n",
+    "Masks are represented by black markers"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "e1201c5b-2308-4dec-a32a-674c7ece40c0",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "da = pp.data.data_array()\n",
+    "da.masks['negative'] = da.data < sc.scalar(0, unit='m/s')\n",
+    "pp.plot(da)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "051ab703-6b08-4ec8-82f6-d2bfc5185359",
+   "metadata": {},
+   "source": [
+    "or a thick black line in the case of bin-edges"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "88af72d3-5e6b-47c8-ac11-d2919455ecc5",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "da = pp.data.data_array(binedges=True)\n",
+    "da.masks['negative'] = da.data < sc.scalar(0, unit='m/s')\n",
+    "pp.plot(da)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "a87c40bc-5ab6-4310-a476-f8e5f5c0c16d",
+   "metadata": {},
+   "source": [
+    "## Using a non-dimension coordinate\n",
+    "\n",
+    "By default, Plopp will use the dimension coordinates to set the horizontal position of the points ad the axis labels.\n",
+    "To use a non-dimension coordinate instead, use the `coords` argument."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "bfcd768c-cc4f-4973-ba55-30c4b7da2d73",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "da = pp.data.data_array()\n",
+    "da.coords['x2'] = da.coords['x']**2\n",
+    "pp.plot(da, coords=['x2'])"
```

Comment:
Maybe be more didactic to just make the data array? Should not be too much code, or is it?